librecad/src/lib/gui/rs_graphicview.cpp

```diff
@@ -1374,8 +1374,19 @@ void RS_GraphicView::drawRelativeZero(RS_Painter *painter) {
     painter->drawLine(RS_Vector(vp.x, vp.y-zr),
                       RS_Vector(vp.x, vp.y+zr)
                       );
-
-    painter->drawCircle(vp, 5);
+    painter->drawLine(RS_Vector(vp.x-zr, vp.y+zr),
+                    RS_Vector(vp.x+zr, vp.y+zr)
+                      );
+    painter->drawLine(RS_Vector(vp.x-zr, vp.y-zr),
+                    RS_Vector(vp.x+zr, vp.y-zr)
+                      );
+    painter->drawLine(RS_Vector(vp.x-zr, vp.y-zr),
+                      RS_Vector(vp.x-zr, vp.y+zr)
+                      );
+    painter->drawLine(RS_Vector(vp.x+zr, vp.y-zr),
+                      RS_Vector(vp.x+zr, vp.y+zr)
+                      );
+    /**painter->drawCircle(vp, 5);*/
```

Comment:
This needed 4 commits.
Think twice and try to test more carefully before committing.